tavern_server/src/db.rs

```diff
@@ -23,6 +23,10 @@ mod tests {
         assert!(conn_string.contains("user=foo"));
         assert!(conn_string.contains("password=bar"));
     }
+    #[test]
+    fn postgres_database_init() {
+        assert!(true);
```

Comment:
As future reference, it's better to make an unimplemented test fail than succeed. Also, the `unimplemented!()` macro exists for this.